web/style/transvision.css

```diff
@@ -578,6 +578,37 @@ fieldset {
     width: 150px;
 }
 
+/* Autocomplete styles for suggestion popup */
+.autocomplete-suggestions {
+    border: 1px solid #999;
+    background: #FFF;
```

Comment:
HEX colors should be all lowercase for consistency, also `background-color` here is enough